zerver/tests/test_queue_worker.py

```diff
@@ -94,6 +87,12 @@ def test_slow_queries_worker(self) -> None:
             with simulated_queue_client(lambda: fake_client):
                 try:
                     worker.setup()
+                    # We call `write_log_line` here because it tests the actual
+                    # path where we publish slow queries to the queue.
+                    with patch('zerver.middleware.is_slow_query', return_value=True):
+                        write_log_line(log_data=dict(test='data'), email='test@zulip.com',
+                                       remote_ip='?', client_name='?', path='/test/',
```

Comment:
I would do `website` for `client_name` and `127.0.0.1` for `remote_ip` to have a more realistic log line.  And `GET` for method.

Reply:
Cool, updated:
```
Running zerver.tests.test_queue_worker.WorkerTest.test_slow_queries_worker
2019-08-17 01:27:36.283 INFO [] Slow query: 127.0.0.1       GET     200 -1000ms (db: 1ms/13q) /test/ (test@zulip.com via website) (test@zulip.com)
```